release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -934,6 +934,17 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use short-time security credentials that are managed outside of the cluster. 
++
+Before this update, when {product-title} was deployed on Azure clusters with Azure AD Workload Identity, storage accounts that were created for the cluster and the image registry had storage account key access enabled by default. The use of storage account keys like Shared Access Signatures (SAS) could create security risks for some {product-title} on Azure deployments.
++
+With this update, when cluster administrators create a new storage account, shared access keys are disabled by default for image registry storage accounts that use Azure AD Workload Identity. This change enhances security by preventing the use of shared access keys.
```

Comment:
Do you think we need to mention here that shared access keys are disabled by default _on new installations_? I see that the paragraph below sort of covers that by mentioning `existing storage accounts` 🤔

Reply:
@flavianmissi Is it 

"With this update, shared access keys are disabled by default on new installations that use Azure AD Workload Identity."

or just

With this update, shared access keys are disabled by default on new installations.

?

Reply:
it is:
"With this update, shared access keys are disabled by default on new installations that use Azure AD Workload Identity."